File: cpp/005.cpp
// 2520 is the smallest number that can be divided by each of the numbers from 1 to 10 without any remainder.

// What is the smallest positive number that is evenly divisible by all of the numbers from 1 to 20?

#include <iostream>

int main () {
    const int max = 20;
    for (int i = 1; true; i++) {
        int success = 0;
        for (int k = 1; k <= max; k++) {
            if (i % k == 0) {
                success++;
            } else {
                break;
            }
        }
        if (success == max) {
                std::cout << i << std::endl;
                return 0;
        }
    }
}

File: js/005.js
var success = 0;

for (var i = 1; i > 0; i++) {
    success = 0;
    for (var k = 1; k <= 20; k++) {
        if (i % k == 0) {
            success += 1;
        } else {
            break;
        }
    }
    if (success == 20) {
        console.log(i);
        return;
    }
}
